src/encoded/search.py

```diff
@@ -78,6 +78,16 @@ def search(context, request, search_type=None, return_generator=False, forced_ty
     principals = request.effective_principals
     es = request.registry[ELASTIC_SEARCH]
 
+    # Get static section (if applicable) when searching a single item type
+    if (len(doc_types) == 1) and 'Item' not in doc_types:
+        search_term = 'search-info-header.' + doc_types[0]
+        static_section = request.registry['collections']['StaticSection'].get(search_term)
+        if static_section:
+            item = static_section.model.source['embedded']
+            result['search_header'] = {}
+            result['search_header']['content'] = item['body']
```

Comment:
Should use `item['content']`, which I guess can be different from `body` (confirmed w/ Alex)